### DISCUSSION TITLE
filter by RIC

### DISCUSSION THREAD

**Discussion by thierry3000** [BOSWatch/BW3-Core#86] [Q&A] [answered]
First, let me thank your for this gorgeous project!!!!
I try to implement some alarms via your routing scheme. Unfortunately the documentation lacks some (in my opinion) essential features. How can I filter for certain rics in pocsag? Is there a possibility to do this inside your config files?

Of course I could hack a filter with regex logic inside the python module, but I hope there is some smarter alternative already implemented.

**Comment by Schrolli91** [comment, marked as answer]
Take a look at the modules - especially the regex filters maybe ;-)
https://docs.boswatch.de/modul/regex_filter.html

You can use the modules inside the router configuration - see here for example:
https://docs.boswatch.de/information/router.html
